src/pre_defined_classes/conditionals/simple_programs_with_decision_points/medium/q3/MainTest.java

```diff
@@ -0,0 +1,44 @@
+package pre_defined_classes.conditionals.simple_programs_with_decision_points.medium.q3;
+
+import global.BaseTest;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import java.io.InvalidClassException;
+import java.util.stream.Stream;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "19";
+        return new Clause[]{
+                new StringLiteral("What is your age\\? "),
+                new NewLine(),
+                new StringLiteral(".*", "underAge")
```

Comment:
You can use the Placeholders for this since they usually want tests to be written with the same style.